apps/ledger-live-desktop/src/renderer/components/ConnectEnvsToSentry.tsx

```diff
@@ -36,10 +36,11 @@ export const ConnectEnvsToSentry = () => {
       });
       // if there are features on, we will add them in tags
       const features: { [key in FeatureId]?: boolean } = {};
-      Object.keys(defaultFeatures).forEach(key => {
-        const value = featureFlags.getFeature(key as keyof typeof defaultFeatures);
-        if (key && value && value.enabled !== defaultFeatures[key as FeatureId]!.enabled) {
```

Comment:
Reworked that part because I used `as const` on defaultFeatures and `as FeatureId` is not working here anymore but I think the alternative that I propose is cleaner
I want to improve the feature flag types a bit too in another PR